packages/daemon/test/test-endo.js

```diff
@@ -1071,3 +1071,34 @@ test('evaluate name resolved by lookup path', async t => {
 
   await stop(locator);
 });
+
+test('list special names', async t => {
+  const { promise: cancelled, reject: cancel } = makePromiseKit();
+  t.teardown(() => cancel(Error('teardown')));
+  const locator = makeLocator('tmp', 'list-names');
+
+  await stop(locator).catch(() => {});
+  await reset(locator);
```

Comment:
@rekmarks Are you on reset-rename-purge?

Reply:
Aye, I will sort out the remaining references tomorrow.